frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -257,27 +258,30 @@ class TableInteractive extends Component {
     } = this.props;
 
     ReactDOM.render(
-      <div style={{ display: "flex" }}>
-        {cols.map((column, columnIndex) => (
-          <div className="fake-column" key={"column-" + columnIndex}>
-            {this.tableHeaderRenderer({
-              columnIndex,
-              rowIndex: 0,
-              key: "header",
-              style: {},
-              isVirtual: true,
-            })}
-            {pickRowsToMeasure(rows, columnIndex).map(rowIndex =>
-              this.cellRenderer({
-                rowIndex,
+      <EmotionCacheProvider>
```

Comment:
Wraps everything with `EmotionCacheProvider` otherwise it won't render correctly. Ideally this should be migrated to `createPortal` or something similar to inherit providers but that would be a large change